configure.py

```diff
@@ -1231,6 +1231,7 @@ def find_ninja():
                 'service/topology_mutation.cc',
                 'service/topology_coordinator.cc',
                 'service/view_building_state.cc',
+                'service/view_building_coordinator.cc',
```

Comment:
Any reason why this new file can't be in db/view instead of service/?

Reply:
moved